cloudify_hostpool_plugin/tasks.py

```diff
@@ -1,4 +1,8 @@
-# #######
+'''
```

Comment:
What docstrings? The "license" in the comments at the top of the file is not a docstring.  What I put in is a docstring that's compatible with RTD.  https://www.python.org/dev/peps/pep-0257/